yolov8_ros/yolov8_ros/yolov8_ros/yolov8_node.py

```diff
@@ -249,6 +287,8 @@ def image_cb(self, msg: Image) -> None:
             if results.boxes:
                 hypothesis = self.parse_hypothesis(results)
                 boxes = self.parse_boxes(results)
```

Comment:
Can you remove all of the code relate to the boxes? Anything that we don't need because we are replacing with iceberg boxes.

Reply:
Keeping old publisher for visualization purposes. Reverted to commit 744ad748b19208b1077d615650d5b2944bdaa700.